### PR TITLE
SL-157/fix/unirest-php-requests

### PR DESCRIPTION
**Pull request by MantasSmulkstys**
https://invertus.atlassian.net/browse/SL-157

Installed new apimatic/unirest-php api library, removed old one.
Rewrote API request get,post methods using unirest syntax.

### CODE REVIEW

File: src/Api/ApiRequest.php [hunk 1]
```diff
@@ -91,4 +88,34 @@ public function get($url, $params = [])
...
+
+    private function getBaseUrl()
+    {
+        return SaferPayConfig::getBaseApiUrl();
+    }
+
+    private function isValidResponse(Response $response)
+    {
+        #TODO: add new exception throw
+        return 1;
```
Comment (tomjas1997) [inline src/Api/ApiRequest.php:117]: can remove this return value.

File: src/Api/Request/InitializeService.php [hunk 2]
```diff
@@ -50,11 +53,8 @@ public function initialize(InitializeRequest $initializeRequest, $isBusinessLice
         }
         $response = $this->apiRequest->post(
             $initializeApi,
-            [
-                'body' => json_encode($initializeRequest->getAsArray()),
-            ]
+            json_encode($initializeRequest->getAsArray()),
```
Comment (tomjas1997) [inline src/Api/Request/InitializeService.php:56]: is 'body' not required key?

Comment (MantasSmulkstys) [reply src/Api/Request/InitializeService.php:56]: maybe for guzzle we needed to add body key that has json, but for unirest it just accepts the json itself.
TLDR: didn't work with body key, works without it

File: src/Api/ApiRequest.php (hunk 1 above)
Comment (MantasSmulkstys) [reply src/Api/ApiRequest.php:117]: will fix this in the next PR

**Review by tomjas1997 — APPROVED** [review]
LGTM

File: src/Api/ApiRequest.php [hunk 3]
```diff
@@ -50,14 +39,16 @@ public function __construct(HttpClientFactory $clientFactory)
      * @return array |null
      * @throws \Exception
      */
-    public function post($url, $params = [])
+    public function post($url, $params)
```
Comment (StanislovasRamanauskas) [inline src/Api/ApiRequest.php:42]: params should be optional
